features/admin.server-configurations.v1/forms/password-recovery-form/password-recovery-form.tsx

```diff
@@ -0,0 +1,801 @@
+                "passwordRecovery.form.fields.smsOtpExpiryTime.validations.range");
+        } else if (!values.smsOtpLength) {
+            // Check for required error
+            errors.smsOtpLength = t("extensions:manage.serverConfigurations.accountRecovery." +
+            "passwordRecovery.form.fields.smsOtpLength.validations.empty");
+        } else if (parseInt(values.smsOtpLength, 10) < GovernanceConnectorConstants
+            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.SMS_OTP_CODE_LENGTH_MIN_VALUE ||
+            parseInt(values.smsOtpLength, 10) > GovernanceConnectorConstants
+                .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.SMS_OTP_CODE_LENGTH_MAX_VALUE) {
+            // Check for invalid input length.
+            errors.smsOtpLength = t("extensions:manage.serverConfigurations.accountRecovery." +
+                "passwordRecovery.form.fields.smsOtpLength.validations.maxLengthReached");
+        } else if (!values.maxResendCount) {
+            // Check for required error
+            errors.smsOtpLength = t("extensions:manage.serverConfigurations.accountRecovery." +
+            "passwordRecovery.form.fields.maxResendCount.validations.empty");
+        } else if (parseInt(values.maxResendCount, 10) < GovernanceConnectorConstants
+            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.MAX_RESEND_COUNT_MIN_VALUE ||
+            parseInt(values.maxResendCount, 10) > GovernanceConnectorConstants
+                .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.MAX_RESEND_COUNT_MAX_VALUE) {
+            // Check for invalid range.
+            errors.maxResendCount = t("extensions:manage.serverConfigurations.accountRecovery." +
+                "passwordRecovery.form.fields.maxResendCount.validations.range");
+        } else if (!values.maxFailedAttemptCount) {
+            // Check for required error
+            errors.maxFailedAttemptCount = t("extensions:manage.serverConfigurations.accountRecovery." +
+            "passwordRecovery.form.fields.maxFailedAttemptCount.validations.empty");
+        } else if (parseInt(values.maxFailedAttemptCount, 10) < GovernanceConnectorConstants
+            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.MAX_FAILED_ATTEMPT_COUNT_MIN_VALUE ||
+            parseInt(values.maxFailedAttemptCount, 10) > GovernanceConnectorConstants
+                .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.MAX_FAILED_ATTEMPT_COUNT_MAX_VALUE) {
+            // Check for invalid range.
+            errors.maxFailedAttemptCount = t("extensions:manage.serverConfigurations.accountRecovery." +
+                "passwordRecovery.form.fields.maxFailedAttemptCount.validations.range");
+        }
+
+        return errors;
+    };
+
+    /**
+     * Prepare form values for submitting.
+     *
+     * @param values - Form values.
+     * @returns Sanitized form values.
+     */
+    const getUpdatedConfigurations = (values: Record<string, any>) => {
+        const data: {
+            "Recovery.ExpiryTime": any;
```

Comment:
move the data const into an interface

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6418/commits/0dd77c8c16b11b96a91e6b1331294e3bb8560a5b